IntBuilder.js

```diff
@@ -0,0 +1,66 @@
+import BuilderES6 from './BuilderES6.js'
+
+// ES6 Way
+export default class IntBuilder extends BuilderES6 {
+  constructor (operand) {
+    super(operand ?? 0)
+    IntBuilder.#throwErrorIfValueNotAnInteger(this._currentResult)
+    this._currentResult = +this._currentResult
+  }
+
+  plus (...values) {
+    for (const value of values) {
+      IntBuilder.#throwErrorIfValueNotAnInteger(value)
+    }
+    return super.plus(...values)
+  }
+
+  minus (...values) {
+    for (const value of values) {
+      IntBuilder.#throwErrorIfValueNotAnInteger(value)
+      this._currentResult -= +value
```

Comment:
Why do you need a plus sign before the `value`?

Reply:
got rid of that